core/lib/env_config/src/fri_prover.rs

```diff
@@ -6,6 +6,7 @@ impl FromEnv for FriProverConfig {
     fn from_env() -> anyhow::Result<Self> {
         let mut prover: FriProverConfig = envy_load("fri_prover", "FRI_PROVER_")?;
         prover.object_store = ObjectStoreConfig::from_env().ok();
+        prover.public_object_store = ObjectStoreConfig::from_env().ok();
```

Comment:
it should be `PublicObjectStore`

Reply:
please don't change `prover.object_store` it will be merged in a few minutes by another pr